deltachat-ios/Coordinator/AppCoordinator.swift

```diff
@@ -768,9 +759,6 @@ class ContactDetailCoordinator: Coordinator, ContactDetailCoordinatorProtocol {
 
     func showEditContact(contactId: Int) {
         let editContactController = EditContactController(dcContext: dcContext, contactIdForUpdate: contactId)
-        let coordinator = EditContactCoordinator(dcContext: dcContext, navigationController: navigationController)
-        childCoordinators.append(coordinator)
-        editContactController.coordinator = coordinator
```

Comment:
4 times the same code duplicated could be removed - one could treat this as a hint, we do not really need this pattern currently.